File: working_on/82612.py
# 부족한 금액 계산하기


def solution(price, money, count):
    total_cost = 0
    for i in range(1, count + 1):
        total_cost += price * i

    answer = money - total_cost
    if answer >= 0:
        return 0
    else:
        return abs(answer)


print(solution(3, 20, 4))

File: working_on/82612.js
// 부족한 금액 계산하기

function solution(price, money, count) {
  var cost = 0;
  for (let i = 1; i <= count; i++) {
    cost += price * i;
  }
  var answer = money - cost;
  if (answer >= 0) {
    return 0;
  } else {
    return answer * -1;
  }
}

console.log(solution(3, 20, 4));
